api/src/main/java/io/opentelemetry/api/common/AttributeKeyImpl.java

```diff
@@ -24,6 +24,11 @@ public String getKey() {
   @Nullable
   abstract String key();
 
+  @Override
+  public final String toString() {
+    return getKey();
```

Comment:
It might be nice to at least delineate that this *isn't* just a String. Maybe put a colon in front of it, so it looks like an elixir atom or a ruby symbol?

Reply:
or after it?

Reply:
I think the type should stay in the toString. Maybe `name(type)`? But I don't hate the current/old string representation.